USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -26,19 +48,52 @@ public string Render(USFMDocument input)
                 output.Append($"<meta charset=\"{encoding}\">");
             }
             output.AppendLine("<link rel=\"stylesheet\" href=\"style.css\">");
-            output.AppendLine("</head>");
-            output.AppendLine("<body>");
 
-            foreach(Marker marker in input.Contents)
+            if (InsertedHTML.Contains("</head>"))
+            {
+                output.AppendLine(InsertedHTML);
+            }
+            else {
+                output.AppendLine("</head>");
+            }
+            
+            // HTML tags can only have one class, when render to docx
+            output.AppendLine($"<body class=\"{(isSingleSpaced ? "" : "double-space")}\">");

```

Comment:
All of these div tags shouldn't be inserted if they don't do any good so you should probably just use if statements around these

Reply:
These are still here. They should be in if statements instead of ternary operators putting in empty divs